spi/transfer-spi/src/test/java/org/eclipse/dataspaceconnector/spi/types/domain/transfer/TransferProcessTest.java

```diff
@@ -139,11 +140,36 @@ void verifyProvisioningComplete() {
     }
 
     @Test
-    void should_pass_from_deprovisioning_request_to_deprovisioning() {
+    void shouldPassFromDeprovisioningRequestToDeprovisioning() {
         TransferProcess process = TransferProcess.Builder.newInstance().id(UUID.randomUUID().toString()).state(DEPROVISIONING_REQ.code()).build();
 
         process.transitionDeprovisioning();
 
         assertThat(process.getState()).isEqualTo(DEPROVISIONING.code());
     }
+
+    @Test
+    void shouldConsideredProvisionedWhenThereAreNoDefinitionsAndNoProvisionedResource() {
```

Comment:
Sorry for being pedantic, but how about:

```java
@Test
@DisplayName("Verifies that provisioning is regarded as complete when there neither ResourceDefinitions nor ProvisionedResources")
void provisionComplete_whenNotDefinitions(){
    // ...
}
```

Reply:
The @DisplayName annotation is a double edged blade because, in fact, is a comment, and as comments it easily becomes obsolete, that's why I used snake_case. However, will use it on those cases.